app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+    has_secure_password
```

Comment:
@idprietop **has_secure_password** could you please explain this?

Reply:
has_secure_password is used to encrypt and authenticate passwords using BCrypt. It assume that the model has a column named password_digest which will be encrypted by BCrypt.

Reply:
@idprietop excellent!